jenkins/databricks/install_deps.py

```diff
@@ -0,0 +1,219 @@
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.arrow--arrow-vector--org.apache.arrow__arrow-vector__*.jar'),
+
+        Artifact('com.google.protobuf', 'protobuf-java',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.google.protobuf--protobuf-java--com.google.protobuf__protobuf-java__*.jar'),
+        Artifact('com.esotericsoftware.kryo', f'kryo-shaded-db',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.esotericsoftware--kryo-shaded--com.esotericsoftware__kryo-shaded__*.jar'),
+        Artifact('org.apache.commons', 'commons-io',
+                 f'{prefix_ws_sp_mvn_hadoop}--commons-io--commons-io--commons-io__commons-io__*.jar'),
+        Artifact('org.apache.commons', 'commons-lang3',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.commons--commons-lang3--org.apache.commons__commons-lang3__*.jar'),
+        Artifact('org.json4s', f'json4s-ast_{scala_version}',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-ast_{scala_version}--org.json4s__json4s-ast_{scala_version}__*.jar'),
+        Artifact('org.json4s', f'json4s-core_{scala_version}',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.json4s--json4s-core_{scala_version}--org.json4s__json4s-core_{scala_version}__*.jar'),
+        Artifact('org.javaassist', 'javaassist',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.javassist--javassist--org.javassist__javassist__*.jar'),
+        Artifact('com.fasterxml.jackson.core', 'jackson-core',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.fasterxml.jackson.core--jackson-databind--com.fasterxml.jackson.core__jackson-databind__*.jar'),
+        Artifact('com.fasterxml.jackson.core', 'jackson-annotations',
+                 f'{prefix_ws_sp_mvn_hadoop}--com.fasterxml.jackson.core--jackson-annotations--com.fasterxml.jackson.core__jackson-annotations__*.jar'),
+        Artifact('org.apache.spark', f'spark-avro_{scala_version}',
+                 f'{spark_prefix}--vendor--avro--avro-*.jar'),
+        Artifact('org.apache.avro', 'avro-mapred',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.avro--avro-mapred--org.apache.avro__avro-mapred__*.jar'),
+        Artifact('org.apache.avro', 'avro',
+                 f'{prefix_ws_sp_mvn_hadoop}--org.apache.avro--avro--org.apache.avro__avro__*.jar'),
+    ]
+
+    # log4j-core
+    if spark_version.startswith('3.3'):
+        deps += Artifact('org.apache.logging.log4j', 'log4j-core',
+                         f'{prefix_ws_sp_mvn_hadoop}--org.apache.logging.log4j--log4j-core--org.apache.logging.log4j__log4j-core__*.jar'),
+
+    return deps
+
+def install_deps(deps, spark_version_to_install_databricks_jars, m2_dir, jar_dir, file):
+    pom_xml_header = """<?xml version=\"1.0\" encoding=\"UTF-8\"?>
+        <project xmlns=\"http://maven.apache.org/POM/4.0.0\"
+            xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"
+            xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\">
```

Comment:
nit: Python """ affords us to stop escaping quotes `\"` ?

```suggestion
    pom_xml_header = """<?xml version="1.0" encoding="UTF-8"?>
        <project xmlns="http://maven.apache.org/POM/4.0.0"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```